src/signals/incident-management/containers/IncidentOverviewPage/components/Map/components/DetailPanel/index.js

```diff
@@ -18,11 +18,17 @@ const Panel = styled.div`
   justify-content: space-between;
 `;
 
+const IncidentLink = styled(AscLink)`
+  font-size: 16px;
+  color: ${themeColor('primary')};
+  font-weight: 700;
```

Comment:
This style rule doesn't seem to be applied:
![Screenshot 2020-04-20 at 09 15 45](https://user-images.githubusercontent.com/1062191/79724272-8940c100-82e7-11ea-944c-087fa1b6cf16.png)

Reply:
Strange, couldn't reproduce it on my development environment. I have checked with the designer and agreed to use a standard asc link here.